tests/unit_tests/test_wave_06.py

```diff
@@ -110,7 +115,20 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result == True
```

Comment:
This check could still pass if `result` were `1`. Using `is` would ensure the `result` really is the boolean `True`.